lib/plotutil.py

```diff
@@ -32,7 +34,14 @@ def plot(func, xstart, xend, step, gui):
 	yvals = create_y_values(func, xvals)
 
 	try:
-		plt.plot(xvals, yvals, linewidth=2.0)
+		#check if color is hex code
+		is_hex = re.search(r'^#(?:[0-9a-fA-F]{3}){1,2}$', color_name)
+		if not is_hex:
+			colors = mcolors.cnames
+			if not color_name in colors:
+				print "Color not found. Setting to default color[Blue]."
+				color_name = 'blue'
+		plt.plot(xvals, yvals, color=color_name,linewidth=2.0)
```

Comment:
please add a space after the **,** present after ```color_name```. Please keep the changes consistent with the code already present as this helps in enhancing code readability :)